src/ConfigPostProcessor.php

```diff
@@ -251,6 +251,10 @@ private function replaceDependencyConfiguration(array $config)
 
         $config = $this->replaceDependencyInvokables($config);
         $config = $this->replaceDependencyFactories($config);
+        $delegators = isset($config['delegators']) ? $this->replaceDependencyDelegators($config['delegators']) : [];
+        if ($delegators) {
+            $config['delegators'] = $delegators;
+        }
```

Comment:
@boesing Just tested a bit, and as we discussed on slack - if it was working in previous version we must have the code which replaces it correctly already - and, this is the simple solution which works as well:

```php
if (isset($config['delegators'])) {
    $config['delegators'] = $this->__invoke($config['delegators'], ['delegators']);
}
```

then we do not need any custom function for delegators.
I think we can even increase this functionality easy later for changes from #63.